apps/remix-ide/src/app/tabs/locales/en/udapp.json

```diff
@@ -109,6 +109,7 @@
 
   "udapp._comment_universalDappUI.tsx": "libs/remix-ui/run-tab/src/lib/components/universalDappUI.tsx",
   "udapp.tooltipTextRemove": "Remove from the list",
+  "udapp.tooltipTextEdit": "Create Dapp with QuickDapp in the main panel",
```

Comment:
Maybe it would be better to have:
Create a DApp using this contract in the main panel.